package.json

```
@@ -4,6 +4,7 @@
   "repository": "github:internetarchive:openlibrary",
   "license": "AGPL-3.0",
   "scripts": {
+    "watch": "webpack -w --mode=development",
```

Let's enable progress; that makes it much easier to see what's  happening. Also write out the full variable name for clarity.

```suggestion
    "watch": "webpack --watch --progress --mode=development",
```